README.md

```diff
@@ -151,20 +151,19 @@ You should now be able to create a new branch, make changes, and open a pull req
 
 #### Add circle.yml to your project
 
-Add the following to your `circle.yml`:
+Add the following to your `.circleci/config.yml`:
 
 ```yml
-dependencies:
-  override:
-    - yarn
-machine:
-  node:
-    version: 6.9
-deployment:
-  appr:
-    branch: /.*/
-    commands:
-      - 'if [ "$CI_PULL_REQUEST" != "" ]; then yarn appr; fi'
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:8
+
+    steps:
+      - checkout
+      - run: yarn install
```

Comment:
what about caching the yarn and maybe add names for the steps?

Reply:
I've added minimal steps like it currently is.